hw/ip/edn/data/edn.hjson

```diff
@@ -0,0 +1,264 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "edn",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  param_list: [
+    { name: "NumEndPoints",
+      desc: "Number of end points to be supported by this block",
+      type: "int",
+      default: "4",
+      local: "true"
+    },
+    { name: "EndPointBusWidth0",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 0",
+      type: "int",
+      default: "64",
+      local: "true"
+    },
+    { name: "EndPointBusWidth1",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 1",
+      type: "int",
+      default: "32",
+      local: "true"
+    },
+    { name: "EndPointBusWidth2",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 2",
+      type: "int",
+      default: "16",
+      local: "true"
+    },
+    { name: "EndPointBusWidth3",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 3",
+      type: "int",
+      default: "8",
+      local: "true"
+    },
+  ],
+  interrupt_list: [
+    { name: "edn_cmd_req_done"
+      desc: "Asserted when a software csrng request has completed."}
```

Comment:
I assume that "software" here means "triggered by a write to SW_CMD_REQ". Maybe it would be better to say that explicitly? Also, "csrng" should probably be capitalized.

Reply:
Agreed on caps. Direct reference was not done by other blocks, so just being consistent.

Reply:
Resolved by Mark.